BUILD.bazel

```diff
@@ -523,9 +523,20 @@ cc_library(
 )
 
 cc_binary(
-    name = "gcs_client_test",
+    name = "redis_gcs_client_test",
     testonly = 1,
-    srcs = ["src/ray/gcs/client_test.cc"],
+    srcs = ["src/ray/gcs/redis_gcs_client_test.cc"],
+    copts = COPTS,
+    deps = [
+        ":gcs",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
+
+cc_binary(
+    name = "actor_state_accessor_test",
+    testonly = 1,
+    srcs = ["src/ray/gcs/actor_state_accessor_test.cc"],
```

Comment:
Can we only have one `cc_binary` or `cc_test` (code can be put in multiple files)? Otherwise it's going to be a lot later.

Reply:
done

Reply:
Did you push new changes?

Reply:
not yet

Reply:
actually, actor_state_accessor_test could only run in the test script(will start redis server first), so it can't build as a cc_test.

Reply:
It'd be better if we can start redis server in the c++ test. 
So we don't need an extra script to run the test.
But if it's not easy, we can do that in next PR.